Makefile

```diff
@@ -115,4 +115,4 @@ lint:
 
 .PHONY: abigen
 abigen:
-	cd api; mkdir ./internal/abi/${package}; abigen --abi=./resource/abi/${package}.json --pkg=${package} --out=./internal/abi/${package}/${package}.go
\ No newline at end of file
+	mkdir ./pkg/abi/${package}; abigen --abi=./resource/abi/${package}.json --pkg=${package} --out=./pkg/abi/${package}/${package}.go
```

Comment:
do you want to use `mkdir -p` to ignore the error of existing directories?

Reply:
Yes. Thanks for the suggestion.